services/workflows-service/src/workflow/context-changed-webhook-caller.ts

```diff
@@ -0,0 +1,57 @@
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
+/* eslint-disable @typescript-eslint/no-unsafe-return */
+
+import { randomUUID } from 'crypto';
+
+export class ContextChangedWebookCaller {
+  #__webhookConfig: ProcessEnvWebhookConfig;
+  #__httpService;
+
+  constructor(eventEmitter: any, webhookConfig: any, httpService: any) {
```

Comment:
I still dont have a cleat picture in mind about what we win/lose with this approach,

I do know that we will lose nest eventEmitter features like events wildcards (`workflow.*`)

Reply:
fixed